Settings.json

```diff
@@ -0,0 +1,16 @@
+{
+    "jack": false,
+    "sample_rate": 48000,
+    "connections": [
+        {
+            "output_stream": "spotify",
+            "input_stream": "ALSA plug-in [turbo_audio]",
+            "port_connections": "AllInOrder"
+        },
+        {
+            "output_stream": "Firefox",
+            "input_stream": "ALSA plug-in [turbo_audio]",
+            "port_connections": "AllInOrder"
+        }
+    ]
+}
```

Comment:
We can probably keep this in the main file for now, if we do ever provide passwords or tokens, we should add to the gitignore and add an example_settings.json file to show the format and explain the options.

Reply:
I think passwords and token should in a .env file tbh